libr/fs/shell.c

```diff
@@ -270,6 +270,22 @@ R_API int r_fs_shell_prompt(RFSShell* shell, RFS* fs, const char* root) {
 				free (f);
 			}
 			free (s);
+		} else if (!memcmp (buf, "open ", 5)) {
```

Comment:
that causes oobread bug. use r_str_startswith() instead

Reply:
also maybe handy to also support "o " as short command for opening. what do you think?

Reply:
Sounds good. Added